restart-core-containers.sh

```diff
@@ -18,5 +18,3 @@ fi
 
 sudo docker-compose up -d broker db geoserver gs-gateway minio
 sudo docker-compose -f docker-compose-test.yml up -d db-test
-
-bash ./s4e-backend/scripts/seed.sh
```

Comment:
why?

Reply:
It will not work without backend instance and as you go to the upper part of the script you will see that all images are killed, therefore it's unnecessary to call seeding script.